src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CSharpStatementBlockOnTypeFormattingTest.cs

```diff
@@ -3,11 +3,17 @@
 
 using System.Threading.Tasks;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
 {
     public class CSharpStatementBlockOnTypeFormattingTest : FormattingTestBase
     {
+        public CSharpStatementBlockOnTypeFormattingTest(ITestOutputHelper output)
```

Comment:
Had no idea this was even a thing. Dope!

Reply:
I _think_ Test Explorer is making this easier in future by just allowing Console.WriteLine to always show up, but this is the "real" way in XUnit.